src/main/java/teammates/ui/controller/InstructorCourseInstructorEditSaveAction.java

```diff
@@ -55,6 +72,26 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         return result;
     }
 
+    /* This method gets the numbers of instructors that would be made visible to students if
+    changes to the instructor information were successfully processed. Thus, if we are
+    trying to make the instructor false, we need to account for that.
+     */
+    protected int getNumberOfInstructorsDisplayedToStudents(String courseId, boolean currentlyDisplayed,
+                                                            boolean isDisplayedToStudents) {
```

Comment:
This variable may need renaming: as there are two variables involving whether the instructor is displayed to students, it needs to be more clear what this refers to (the old display state of the instructor, or the new one should the changes be processed).